activerecord/lib/active_record/nested_attributes.rb

```diff
@@ -304,9 +304,9 @@ module ClassMethods
       #   # creates avatar_attributes= and posts_attributes=
       #   accepts_nested_attributes_for :avatar, :posts, allow_destroy: true
       def accepts_nested_attributes_for(*attr_names)
-        options = { :allow_destroy => false, :update_only => false }
+        options = { :allow_destroy => false, :update_only => false, :index_errors => false }
```

Comment:
@mprobber - I'm not sure if we went over that at open academy, using the new hash syntax means it should read like this instead:

`options = { allow_destroy: false, update_only: false, index_errors: false }`